justfile

```
@@ -1,6 +1,64 @@
 @_default:
     just --list --unsorted
 
+# Run all recipes
+run-all: install-package-dependencies document run-targets spell-check style lint test build-site check install-package
+
+# Install package dependencies
+install-package-dependencies:
+  #!/usr/bin/Rscript
+  pak::pak(ask = FALSE)
+
+# Run document generators
+document:
+  #!/usr/bin/Rscript
+  devtools::document()
+
+# Run the pipeline to re-build the data objects
+run-targets:
+  #!/usr/bin/Rscript
+  targets::tar_make()
+
+# Run the package tests
+test:
+  #!/usr/bin/Rscript
+  devtools::test()
+
+# Check the spelling
+spell-check:
+  #!/usr/bin/Rscript
+  devtools::spell_check()
+
+# Check URLs based on CRAN requirements
+url-check:
+  #!/usr/bin/Rscript
+  urlchecker::url_check()
+
+# Style all R code in the package
+style:
+  #!/usr/bin/Rscript
+  styler::style_pkg()
+
+# Run the linter to check for things not caught by styler
+lint:
+  #!/usr/bin/Rscript
+  devtools::lint()
+
+# Build the pkgdown website
+build-site:
```

```suggestion
build-website:
```
(just to keep it aligned with what we call the corresponding recipe in Seedcase)